web-common/src/components/forms/InputV2.svelte

```diff
@@ -57,21 +58,21 @@
   {/if}
 
   <input
-    {id}
-    type="text"
-    {placeholder}
-    name={id}
     autocomplete="off"
+    bind:this={inputElement}
+    bind:value
     class:error={error && value}
+    {id}
+    name={id}
+    on:blur={() => (focus = false)}
     on:change
+    on:focus={() => (focus = true)}
     on:input
     on:keydown={handleKeyDown}
-    on:focus={() => (focus = true)}
-    on:blur={() => (focus = false)}
-    bind:this={inputElement}
-    bind:value
+    {placeholder}
+    type="text"
   />
-  {#if error && !focus && value}
+  {#if error && (alwaysShowError || (!focus && value))}
```

Comment:
This is getting complicated. Rather, I think we should:
- Use just `{#if error}`
- At the form-level, validate the field on every keystroke

Reply:
Hmm i was not a 100% sure what the other checks were. So i dont want to unintentionally break other features.